MM-51662 Update help text for settings to enable SVGs and Latex

hmhealey opened the pull request:
#### Summary

We're adding a recommendation for enabling SVGs similar to the one that enabling Latex has. Also, the default values listed for these two settings are incorrect for new installs. The one for inline Latex is correct though since it only has an effect when regular Latex is turned on.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-51662

#### Related Pull Requests
https://github.com/mattermost/mattermost-server/pull/22956

cwarnermm commented on source/configure/site-configuration-settings.rst:
```diff
@@ -1151,33 +1151,35 @@ Enable message link previews
-| - **true**: **(Default)** Enables previews of SVG files attached to messages. | - System Config path: **Site Configuration > Posts**             |
-| - **false**: Disables previews of SVG files.                                  | - ``config.json`` setting: ``.ServiceSettings.EnableSVGs: true`` |
+| - **true**: Enables previews of SVG files attached to messages.               | - System Config path: **Site Configuration > Posts**             |
+| - **false**: **(Default)** Disables previews of SVG files.                    | - ``config.json`` setting: ``.ServiceSettings.EnableSVGs: true`` |
```
Can you please update the example in the right column from ``true`` to ``false`` as well? Thanks!

cwarnermm commented on source/configure/site-configuration-settings.rst:
```diff
@@ -1151,33 +1151,35 @@ Enable message link previews
-| - **true**: **(Default)** Enables rendering of `LaTeX in code blocks <https://docs.mattermost.com/channels/format-messages.html#math-formulas>`__. | - System Config path: **Site Configuration > Posts**              |
-| - **false**: Disables rendering in blocks. Instead, LaTeX code is highlighted.                                                                     | - ``config.json`` setting: ``.ServiceSettings.EnableLatex: true`` |
+| - **true**: Enables rendering of `LaTeX in code blocks <https://docs.mattermost.com/channels/format-messages.html#math-formulas>`__.               | - System Config path: **Site Configuration > Posts**              |
+| - **false**: **(Default)** Disables rendering in blocks. Instead, LaTeX code is highlighted.                                                       | - ``config.json`` setting: ``.ServiceSettings.EnableLatex: true`` |
```
Can you please update the example in the right column from true to false as well? Thanks!

cwarnermm reviewed (commented):
Thanks, @hmhealey, for this docs PR. I've identified a couple of small updates needed.

hmhealey commented:
@cwarnermm I updated those, fixed the missing asterisks from the end of "warning", and also changed the warning on the Latex settings to match the SVG one since that's what we did in the System Console

esarafianou reviewed (approved):
LGTM!
